### PR TITLE
Sublime Text 3.2 git integration documentation.

### PR DESCRIPTION
**Pull request by HonkingGoose**
Hi @ben,

I've added a section on Sublime Text. This pull-request would help with a part of issue #1205.

To clarify: you can't do any git operations from within the text editor alone, that's reserved for the Sublime Merge git client. There are hooks into Sublime Merge from within Sublime Text so that some git commands can be done, this requires that Sublime Merge is installed of course.

Maybe a section on Sublime Merge would be handy in the "Graphical Interfaces" part of the Git Pro book? Maybe Gitkraken can be mentioned in that same section? It might be duplication of effort, there are lists of git gui interfaces out there on the web. So I'll leave that up to you to decide. :+1: 

---

Hi @wbond,

Thanks for making Sublime Text 3, and thanks for making the git integration in the editor. I like it, it make things a lot clearer. :+1: 

As you can see, if written some text to go into the Git Pro book. I would like your feedback on this text, because you're the developer of the program.

Also this way you can give your blessing/disapproval to inclusion of text about Sublime Text into this book.

---

Greetings HonkingGoose

### CODE REVIEW

File: A-git-in-other-environments.asc [hunk 1]
```diff
@@ -13,13 +13,14 @@ include::book/A-git-in-other-environments/sections/visualstudio.asc[]
...
 
-
 include::book/A-git-in-other-environments/sections/bash.asc[]
 
 include::book/A-git-in-other-environments/sections/zsh.asc[]
 
 include::book/A-git-in-other-environments/sections/powershell.asc[]
 
+included::book/A-git-in-other-environments/sections/sublimetext.asc[]
+
```
Comment (ben) [inline A-git-in-other-environments.asc]: Can you move this up to just below the Eclipse section?

File: book/A-git-in-other-environments/sections/sublimetext.asc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+=== Git in Sublime Text
+
+From version 3.2 onwards, Sublime Text has git integration in the editor.
+
+The features are:
+*  The sidebar will show the git status of files and folders with a badge/icon.
+*  Files and folders that are in your .gitignore file will be faded out in the sidebar.
+*  In the status bar, you can see the current git branch and how many modifications you have made.
+*  All changes to a file are now visible via markers in the gutter.
+*  You can use part of the Sublime Merge git client functionality from within Sublime Text. (This requires that Sublime Merge is installed.)
```
Comment (ben) [inline book/A-git-in-other-environments/sections/sublimetext.asc]: Two things:

1. Let's move the parenthetical to a new line to keep with our source code conventions.
1. Can you link "Sublime Merge" to [its marketing site](https://www.sublimemerge.com/)?

File: book/A-git-in-other-environments/sections/sublimetext.asc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+The official documentation can be found here: https://www.sublimetext.com/docs/3/git_integration.html
```
Comment (ben) [inline book/A-git-in-other-environments/sections/sublimetext.asc]: If you add `[]` to the end of this line it'll become a link.

File: A-git-in-other-environments.asc (hunk 1 above)
Comment (HonkingGoose) [reply A-git-in-other-environments.asc]: Done.

File: book/A-git-in-other-environments/sections/sublimetext.asc (hunk 2 above)
Comment (HonkingGoose) [reply book/A-git-in-other-environments/sections/sublimetext.asc]: Done.

File: book/A-git-in-other-environments/sections/sublimetext.asc (hunk 3 above)
Comment (HonkingGoose) [reply book/A-git-in-other-environments/sections/sublimetext.asc]: Fixed link.

**Comment by ben** [comment]
🌈 Thanks!

File: book/A-git-in-other-environments/sections/sublimetext.asc (hunk 2 above)
Comment (HonkingGoose) [reply book/A-git-in-other-environments/sections/sublimetext.asc]: Was this newline how you wanted it? I'm unsure, asciidoctor's documentation wasn't clear on a newline after a bullet point...

Comment (ben) [reply book/A-git-in-other-environments/sections/sublimetext.asc]: This is a bit odd. We use this markup style in [this other section](https://github.com/progit/progit2/blob/15c51bb92dba00b14d343ac52a913ff89f07ff6a/book/07-git-tools/sections/credentials.asc#L17) and it renders fine, but here the GitHub renderer is choking on the formatting. Not sure what the difference is.

**Comment by ben** [comment]
✨

**Pull request merged**